layouts/joomla/sidebars/submenu.php

```diff
@@ -18,9 +18,9 @@
 <?php if ($displayData->displayMenu || $displayData->displayFilters) : ?>
 <div id="j-toggle-sidebar-wrapper">
 	<div id="sidebar" class="sidebar">
-		<button class="btn btn-sm btn-secondary my-2 options-menu d-md-none" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="<?php echo Text::_('TPL_SPRING_TOGGLE_SIDEBAR'); ?>">
+		<button class="btn btn-sm btn-secondary my-2 options-menu d-md-none" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="<?php echo Text::_('TPL_KHONSU_TOGGLE_SIDEBAR'); ?>">
```

Comment:
You should not use a template specific language string in a core non template file. See https://github.com/joomla/joomla-cms/pull/27082 for reference